### DISCUSSION TITLE
https://blog.moroz.cc/post/arch-linux-install-guide-uefi--encrypted-lvm/

### DISCUSSION THREAD

**Discussion by giscus** [blogmorozcc/blogmorozcc.github.io#20] [General]
# https://blog.moroz.cc/post/arch-linux-install-guide-uefi--encrypted-lvm/

Introduction This is my own guide for installing Arch Linux on bare metal machine with UEFI, encrypted LVM and separate /home partition.
Steps First you need to create and boot the installation media on your PC, as the result you will boot into plain console.
Increase font size As most modern laptops/PCs have large resolution displays I recommend increasing the font size:
setfont ter-132b Setup the internet connection In this example I have a laptop with Wi-Fi modem, so I&rsquo;ll be using iwd to setup the internet connection.

https://blog.moroz.cc/post/arch-linux-install-guide-uefi--encrypted-lvm/

**Comment by smahm006** [comment]
Ty for the guide helped me a lot!
